Sources/Common/Core/Math/index.js

```diff
@@ -2135,33 +2138,25 @@ export function float2CssRGBA(rgbArray) {
 }
 
 export function rotateVector(vectorToBeRotated, axis, angle) {
```

Comment:
it would be great if you can create test to make sure that the problem does not come up in the future (e.g. if we decide to no longer use vtkMatrixBuilder). 
This was a hard to find bug, so it would be a hard to find regression.